home-assistant-plugin/custom_components/light/xknx.py

```diff
@@ -197,42 +204,12 @@ def color_temp(self):
     @property
     def min_mireds(self):
         """Return the coldest color temp this light supports in mireds."""
-        if self.device.supports_color_temperature or \
-                self.device.supports_tunable_white:
-            kelvin = self.device.max_kelvin
-            return color_util.color_temperature_kelvin_to_mired(kelvin) \
-                if kelvin is not None else DEFAULT_MIN_MIREDS
-        return None
+        return self._min_mireds
 
     @property
     def max_mireds(self):
         """Return the warmest color temp this light supports in mireds."""
-        if self.device.supports_color_temperature or \
-                self.device.supports_tunable_white:
-            kelvin = self.device.min_kelvin
-            return color_util.color_temperature_kelvin_to_mired(kelvin) \
-                if kelvin is not None else DEFAULT_MAX_MIREDS
-        return None
-
-    @property
-    def min_kelvin(self):
```

Comment:
I assume we removed this kelvin* properties due to the fact that HA doesn't support it right?

Reply:
yes! No need to override. Instance variable is enough since its not going to change at runtime. 
If HA decides to support Kelvin too we can always make it a property then.

Reply:
Why do we need the kelvin variables at all then? :)